app/workers/search/index_worker.rb

```diff
@@ -0,0 +1,18 @@
+module Search
+  class IndexWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :default, retry: 10
+
+    def perform(record_type, record_id)
+      raise InvalidRecordType unless %w[Comment Article User PageView].include?(record_type)
```

Comment:
I know you just moved over this job, but since we're already refactoring, can you please pull the array out of `perform` and assign it to a constant, e.g. 

```ruby
ValidRecordTypes = %w[Comment Article User PageView].freeze
```

and then

```diff
- raise InvalidRecordType unless %w[Comment Article User PageView].include?(record_type)
+ raise InvalidRecordType unless ValidRecordTypes.include?(record_type)
```

Recreating it on every invocation of `perform` seems a bit unnecessary.